test/testcases/kmesh/libs/common.sh

```diff
@@ -12,6 +12,11 @@ function env_init()
     yum localinstall -y fortio-*$(arch).rpm
     cd $CURRENT_PATH
 
+    cd /sys/fs/cgroup/net_cls && mkdir -p kmesh_test && cd kmesh_test
```

Comment:
cgroup path needs to be deleted after the test case execution is complete.

Reply:
OK, add 'cgdelete net_cls:/kmesh_test' in cleanup